src/main/java/clients/symphony/api/constants/DatafeedVersion.java

```diff
@@ -0,0 +1,13 @@
+package clients.symphony.api.constants;
+
+public enum DatafeedVersion {
```

Comment:
do we need to expose this enum publicly in the SDK?

Reply:
I expose this enum to use it in classes from other package, ex: DatafeedClient, DatafeedEventsService